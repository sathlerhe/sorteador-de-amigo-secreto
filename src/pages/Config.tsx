import Footer from "../components/Footer";
import Form from "../components/Form";
import ParticipantsList from "../components/ParticipantsList";

export default function Config() {
  return (
    <>
      <Form />
      <ParticipantsList />
      <Footer />
    </>
  )
}