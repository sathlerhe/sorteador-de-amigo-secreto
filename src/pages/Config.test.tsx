import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Config from "./Config";

const mockNavigate = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavigate,
  };
});

describe("Config.tsx behavior", () => {
  it("Should render correctly", () => {
    const { container } = render(
      <RecoilRoot>
        <Config />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
