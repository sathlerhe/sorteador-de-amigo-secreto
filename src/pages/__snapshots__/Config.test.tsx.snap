// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config.tsx behavior Should render correctly 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
